Makefile.standalone

```diff
@@ -0,0 +1,52 @@
+
+## path configuration (FHS-style)
+PREFIX ?= /usr
+BINDIR ?= $(PREFIX)/bin
+SBINDIR ?= $(PREFIX)/sbin
+
+## toolchain commands
+export GO ?= go
+export INSTALL ?= install
+export LN_S ?= ln -s
+export MKDIR_P ?= mkdir -p
+
+BUILD_TMP ?= $(CURDIR)/tmp
+BUILD_OUT ?= $(BUILD_TMP)/build
+export GOCACHE ?= $(BUILD_TMP)/go-cache
+export LDFLAGS=-s -w
+
+BUILD_GO_ARCH := $(shell $(GO) env GOARCH | sed -e 's~^386~i386~;')
+BUILD_GO_OS := $(shell $(GO) env GOOS)
+
+EXEC_NAME=dockerd-$(BUILD_GO_OS)-$(BUILD_GO_ARCH)
+EXEC_OUT=$(BUILD_OUT)/$(EXEC_NAME)
+
+# make sure it fails ...
+export GOPATH=/proc/gopath
+export GOPROXY=127.0.0.1:29999
```

Comment:
This should be in your own .bash_profile

Reply:
> This should be in your own .bash_profile

No, this is an extra measure to prevent go from ever getting anything downloaded. See above.
Otherwise build completely breaks. And even it it wouldn't, we wouldn't have reproducable build (thus not auditable).

This is not an environment issue (actually, environment should not and cannot take care of such package specific details), but a way to make the build process work at all.